config/initializers/feedjira.rb

```diff
@@ -1,3 +1,5 @@
 load "#{Rails.root}/lib/feedjira/parser/podcast.rb"
 
-Feedjira::Feed.add_feed_class Feedjira::Parser::Podcast
+Feedjira.configure do |config|
+  config.parsers.unshift(Feedjira::Parser::Podcast)
```

Comment:
`add_feed_class` no longer exists... but following [the README](https://github.com/feedjira/feedjira/tree/main#parsers)... this "just worked".

I'm not entirely clear if this is a change.  Did `add_feed_class` push the parser on the _end_ of the list, or the beginning?

In any case - the tests still pass.  So... success?